12_glob_lab/globlab.txt

```diff
@@ -3,27 +3,36 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
 1. Type in a glob pattern to match:
 
-  1. All files whose name starts with a lowercase letter
+  
+1. All files whose name starts with a lowercase letter
   Answer: 
 
-  2. All files whose name starts with an uppercase letter
+  echo [a-z]*
+2. All files whose name starts with an uppercase letter
   Answer:
 
-  3. All files whose name ends in an "a", "e", "i", "o" or "u"
+  LC_ALL=C bash -c 'echo [A-Z]*'

```

Comment:
Cool. I'd use `export LC_ALL=C` before doing the exercise which affects all commands in a session